### PR TITLE
chore: setup docker to run the application with database as container

### PR DESCRIPTION
**Pull request by mittmannv8**
Enable to use the Postgres database with docker container.

Closes (#67)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,17 +6,19 @@ help:
...
 	@echo "run-python ................................ Run just backend side"
 	@echo "setup ..................................... Setup the whole project for development purpose"
 	@echo "setup-database ............................ Configure the database for development purpose"
 	@echo "setup-dependencies ........................ Install frontend and backend dependencies"
 	@echo "migrate ................................... Run the database migration"
 	@echo "test ...................................... Run frontend and backend tests"
-	@echo "test-docker ............................... Run frontend and backend tests inside container"
 	@echo "test-javascript ........................... Run the frontend tests"
 	@echo "test-python ............................... Run the backend tests"
+	@echo "docker-migrate ............................ Run the database migration inside container"
```
Comment (kplaube) [inline Makefile]: Can you organise it alphabetically? Thanks :)

File: Makefile [hunk 2]
```diff
@@ -64,11 +64,20 @@ migrate:
...
-test-docker:
-	docker-compose run --rm app sh -c "npm run test & pytest ."
-
 test-javascript:
 	npm run test
 
 test-python:
 	pipenv run test
+
+docker-setup: docker-run docker-migrate
```
Comment (kplaube) [inline Makefile]: To keep consistency, would you mind organising it alphabetically?

Thanks :)

**Comment by kplaube** [comment]
It's approved already. Anything blocking it from merging?

**Comment by mittmannv8** [comment]
> It's approved already. Anything blocking it from merging?

Just

File: Makefile (hunk 2 above)
Comment (mittmannv8) [reply Makefile]: It can be reorgnanised. I thinked about the context for ordering.

**Pull request merged**